pkg/resources/endpoints_test.go

```diff
@@ -116,12 +127,14 @@ func endpoints(ipCount int) *corev1.Endpoints {
 			Name:      testService,
 		},
 	}
-	addresses := make([]corev1.EndpointAddress, ipCount)
+	addresses, notReadyAddresses := make([]corev1.EndpointAddress, ipCount), make([]corev1.EndpointAddress, ipCount)
```

Comment:
let's use 2 params and different numbers -- otherwise there might be a sneaky regression.

Reply:
done